twake/backend/node/test/e2e/users.spec.ts

```diff
@@ -40,15 +43,109 @@ describe.skip("The /users API", () => {
         },
       });
 
-      expect(response.statusCode).toBe(200);
+      expect(response.statusCode).toBe(404);
       expect(response.json()).toEqual({
-        id,
+        error: "Not Found",
+        message: `User ${id} not found`,
+        statusCode: 404,
+      });
+      done();
+    });
+
+    it("should 200 and big response for myself", async done => {
+      const id = "9ba49092-af64-11eb-90a9-0242ac150004";
```

Comment:
I suppose keeping this hard coded will break the ci tests ?

Reply:
Yep, it is…   gonna fix this